omnibus/cookbooks/omnibus-supermarket/libraries/config.rb

```diff
@@ -46,7 +46,15 @@ def self.load_or_create_secrets!(filename, node)
       node.consume_attributes('supermarket' => secrets)
     rescue Errno::ENOENT
       begin
-        secrets = { 'secret_key_base' => SecureRandom.hex(50) }
+        secret_key_base = if node['supermarket'] && node['supermarket']['secret_key_base']
```

Comment:
Could simplify this a bit by using `dig`
```
secrets_key_base = if node.dig('supermarket', 'secret_key_base')
```

Reply:
Mmmm. `.dig`. Nice. 🤔

Reply:
I tried `node.dig`. Tests passed in my development environment, but didn't in CI. I think I'm at the limit of my rabbit hole timebox for testing this, so Imma opt for ugly `&&` check.